gmso/core/topology.py

```diff
@@ -790,12 +790,12 @@ def get_untyped(self, group):
         }
         if group == "topology":
             for subgroup in untyped_extractors:
-                untyped.update(untyped_extractors[subgroup])
-        elif group in untyped_extractors:
-            untyped = untyped_extractors[group]()
+                untyped.update(untyped_extractors[subgroup]())
         elif isinstance(group, (list, tuple, set)):
             for subgroup in group:
                 untyped.update(untyped_extractors[subgroup]())
+        elif isinstance("group", str) and group in untyped_extractors:
```

Comment:
What is the reason where you are checking `"group"` as a string?

Reply:
oops, that's a typo

Reply:
fixed